tcms/testruns/views.py

```diff
@@ -36,25 +36,34 @@ class CreateTestRunView(View):
     """Display the create test run page."""
 
     template_name = 'testruns/mutable.html'
-    http_method_names = ['post']
+    http_method_names = ['post', 'get']
 
-    def post(self, request):
-        # If from_plan does not exist will redirect to plans for select a plan
-        plan_id = request.POST.get('from_plan')
+    def get(self, request):
```

Comment:
IMO the easiest way would be to split the view change into a separate pull request and make sure all tests referencing this view pass before continuing further.